kube-controllers/pkg/controllers/loadbalancer/loadbalancer_controller.go

```diff
@@ -671,13 +671,25 @@ func (c *loadBalancerController) assignIP(svc *v1.Service) ([]string, error) {
 		return nil, nil
 	}
 
+	// Get the namespace object for namespaceSelector support
+	var namespaceObj *v1.Namespace
+	if svc.Namespace != "" {
+		ns, err := c.clientSet.CoreV1().Namespaces().Get(context.Background(), svc.Namespace, metav1.GetOptions{})
```

Comment:
```suggestion
		namespaceObj, err = c.clientSet.CoreV1().Namespaces().Get(context.Background(), svc.Namespace, metav1.GetOptions{})
```

Same here - we can assign directly.